frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -42,22 +40,37 @@ const QueriesMatchingCapacityView: QueryMapType = {
     CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_REQUESTED_QUERY],
     CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_USED_QUERY],
   ],
-  [CapacityViewType.CAPACITY_VMS_VS_PODS]: [
-    CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_PODS_QUERY],
-    CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_VMS_QUERY],
-  ],
 };
 
-export const CapacityCard: React.FC<DashboardItemProps> = ({
+const getItems = (plugin: any, flags: FlagsObject) =>
+  plugin.filter((e) => flags[e.properties.required]);
+
+const getCapacityQueries = (flags) => {
```

Comment:
Add types for flag which I think will be `FlagsObject` only

Reply:
ack